### DISCUSSION TITLE
Github Developer Pack failing application

### DISCUSSION THREAD

**Discussion by Muiruri590** [community/community#125166] [GitHub Education]
### Hi everyone,

Hi everyone , when i try submitting my documents i keep getting this error "Please upload another picture of your academic proof.  and you don't appear to be within the location of the school you have chosen " even though i have uploaded more than three documents now  and they all have specific dates for expiry , and i have allowed my browser to allow location access 
and then am not within the school compound but am within my country .
someone kindly help

**Comment by ankit-orion** [comment]
I understand how frustrating this must be. Here are a few steps you can try to resolve this issue:

Double-Check Your Documents: Ensure the documents you are uploading clearly show your academic proof and that they are not expired. Make sure the documents are in the required format (e.g., PDF, JPEG) and are clear and legible.

Verify Location Settings: Ensure your browser and device have accurate location settings enabled. You can do this by:

Checking if location services are enabled in your device settings.
Ensuring your browser has location permissions enabled for the website you are using.
Use a Different Browser or Device: Sometimes, switching to a different browser or device can resolve technical issues. Try accessing the submission portal from another browser or device to see if the issue persists.

Clear Browser Cache: Clear your browser’s cache and cookies to remove any corrupted data that might be causing the issue. This can often resolve unexpected errors on websites.

Contact Support: If the problem continues, contact the support team for the platform you are using. Provide them with detailed information about the error message, the documents you have uploaded, and the steps you have taken so far. They may be able to offer specific guidance or resolve the issue on their end.

**Reply by govind848132** [reply]
+1

**Comment by Muiruri590** [comment]
now i sent my application and i got this

   - "We have already received a discount request for @Muiruri590. We will
   get back to you soon." somehow i suspect that because the last week i had
   submitted my application but i have not heard from them so i thought i
   would send again  will it cause any issue , or is there another meaning of
   this statement "
      - We have already received a discount request for @Muiruri590. We
      will get back to you soon."
